guides/source/action_text_overview.md

```diff
@@ -112,6 +112,90 @@ class MessagesController < ApplicationController
 end
 ```
 
+## Rendering Rich Text content
+
+ActionText will sanitize and render rich content on your behalf.
+
+By default, the Action Text editor and content is styled by the Trix defaults.
+
+If you want to change these defaults, you'll want to remove the
+`app/assets/stylesheets/actiontext.scss` linker and base your stylings on the
+[contents of that file](https://raw.githubusercontent.com/basecamp/trix/master/dist/trix.css).
+
+By default, ActionText will render rich text content into an element that
+declares the `.trix-content` class:
+
+```html+erb
+<%# app/views/layouts/action_text/_content.html.erb %>
+<div class="trix-content">
+  <%= yield %>
+</div>
+```
+
+If you'd like to change the rich text's surrounding HTML with your own layout,
+declare your own `layouts/action_text/content` template and call `yield` in
+place of the content.
+
+You can also style the HTML used for embedded images and other attachments
+(known as blobs). On installation, Action Text will copy over a partial to
+`app/views/active_storage/blobs/_blob.html.erb`, which you can specialize.
+
+### Rendering attachments
+
+In addition to attachments uploaded through ActiveStorage, ActionText can embed
+anything that can be resolved by a [Signed GlobalID](https://github.com/rails/globalid#signed-global-ids).
+
+ActionText renders embedded `<action-text-attachment>` elements by resolving
+their `sgid` attribute into an instance. Once
+resolved, that instance is passed along to
+[`render`](https://edgeapi.rubyonrails.org/classes/ActionView/Helpers/RenderingHelper.html#method-i-render).
+The resulting HTML is embedded as a descendant of the `<action-text-attachment>`
+element.
+
+For example, consider a `User` model:
+
+```ruby
+# app/models/user.rb
+class User < ActiveRecord::Base
+  has_one_attached :avatar
+end
+
+user = User.find(1)
+user.to_global_id.to_s #=> gid://MyRailsApp/User/1
+user.to_signed_global_id.to_s #=> BAh7CEkiCG…
+```
+
+Next, consider some rich text content that embeds an `<action-text-attachment>`
+element that references the `User` instance's signed GlobalID:
+
+```html
+<p>Hello, <action-text-attachment sgid="BAh7CEkiCG…"></action-text-content>.</p>
+```
+
+ActionText resolves uses the "BAh7CEkiCG…" String to resolve the `User`
+instance. Next, consider the application's `users/user` partial:
+
+```html+erb
+<%# app/views/users/_user.html.erb %>
+<span><%= image_tag user.avatar %> <%= user.name %></span>
```

Comment:
I think it would be _awesome_ to give users more control over which partial gets rendered when resolving `<action-text-attachment>` elements (like we do with `ActionText::Attachable#to_trix_content_attachment_partial_path`):

https://github.com/rails/rails/blob/11738e82cdcfb48f7740a2da01ad2716d9bcfbb6/actiontext/lib/action_text/attachable.rb#L70-L72

Those changes felt somewhat out of scope. I planned on opening a follow-up pull request to introduce that seam, but I could add it here if it'd be better to treat it as a contiguous changeset.